Deploy/stacks/dynamic/examples/datasets/inputs/config/building-cambridge.sql

```diff
@@ -1,31 +0,0 @@
-WITH "toid_table" AS (
-    SELECT "strval" AS "toid",
-        "cityobject_id"
-    FROM "cityobject_genericattrib"
-    WHERE "attrname" = 'os_topo_toid'
-),
```

Comment:
I note that the SQL file you replaced this one with no longer makes reference to TOIDs. I'm guessing this is for good reasons?

Reply:
It was removed so building-bavaria and building-cambridge can share the same SQL file for geoserver layer publication.